selfdrive/car/hyundai/interface.py

```diff
@@ -144,13 +144,15 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.lateralTuning.pid.kiBP, ret.lateralTuning.pid.kpBP = [[0.], [0.]]
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.05]]
     elif candidate == CAR.KIA_SELTOS:
-      ret.lateralTuning.pid.kf = 0.00005
-      ret.mass = 1310. + STD_CARGO_KG
```

Comment:
why are all these other lines deleted? only remove the ones for the PID tuning

Reply:
The PID lines were already removed, didn't get what you meant by
> delete this, it's already the default
I've reverted the changes, it should have already been correct I would think.